src/scripts.js

```diff
@@ -3,6 +3,9 @@ import apiCalls from './apiCalls';
 import { recipeData } from './data/recipes';
 import RecipeRepository from './classes/RecipeRepository';
 
+// let recipesList = recipesData;  need to plug in recipeData here
```

Comment:
Again, comments should be removed for PRs unless they're providing specific in-code context that would help other devs.